experimental/add_to_app/example_module/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: example_module
-description: A simple module used to demo Flutter's Add2App features
+description: A simple module used to demo Flutter's Add-to-App features
```

Comment:
I could have sworn I grepped the code and fixed all of these. Thanks for catching it.